CONTRIBUTING.md

```diff
@@ -3,9 +3,7 @@ Contribution Agreement
 
 As a contributor, you represent that the code you submit is your original work or
 that of your employer (in which case you represent you have the right to bind your
-employer). By submitting code, you (and, if applicable, your employer) are
-licensing the submitted code to LinkedIn and the open source community subject to
-the BSD 2-Clause license.
+employer). By submitting code, you (and, if applicable, your employer) are licensing the submitted code to LinkedIn and the open source community subject to the BSD 2-Clause license.
```

Comment:
This should be: "By submitting code, you (and, if applicable, your employer) are licensing the submitted code under the Apache 2.0 license."

Reply:
updated!